service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java

```diff
@@ -236,6 +236,13 @@ CaseDetails asCaseDetails(CaseData caseData) {
             .build();
     }
 
+    CallbackRequest toCallBackRequest(CaseDetails caseDetails, CaseDetails caseDetailsBefore) {
```

Comment:
maybe it would be a bit more benefitial to have overloaded method that accepts CaseData as both params